docs/jobs/ai-assistant.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Using OpenFn with AI Assistant
+sidebar_label: Using AI Assistant
+---
+
+:::warning
+
+AI makes mistakes. Always review the AI's response carefully and make sure to
+verify the output. You can help us improve the AI Assistant by providing
+feedback on the response.
+
+:::
+
+## About OpenFn AI Assistant
+
+The AI Assistant is built on the Claude 3.5 Sonnet model from Anthropic and is
+trained with OpenFn documentation and example job code. The goal of the
+assistant is to help you with your jobs and diagnosing issues with your jobs and
+error logs. It is important to note that the assistant is not a replacement for
+human developers. It is a tool to help you with your jobs and diagnosing issues
+with your jobs and error logs.
+
+:::caution
+
+Before you start, we recommend that you take a moment to read our more about the
+AI Assistant and
+[Responsible AI principles](https://www.openfn.org/responsible-ai/) by clicking
+on the `Learn more about AI Assistant` above the button.
+
+:::
+
+## How to access the AI Assistant
+
+The AI Assistant is located in the IDE/Inspector as a new tab in the Input
+panel. To use the Assistant, click on the `AI Assistant` tab. If it is your
+first time, this will open an introductory screen with a button that says
+`Get Started with the AI Assistant`.
+
+![Assistant Introductory Screen](/img/ai-assistant-intro.png)
+
+By clicking on the `Get Started with AI Assistant` button, the chat interface
+will open and you will be able to start chatting with the AI Assistant.
+
+![AI Assistant](/img/ai-chat-empty.png)
+
+If you have had a previous chat session, you will see the list of previous
+sessions listed in cards names with the first prompt of the session.
+![Previous Sessions](/img/ai-assistant-sessions.png)
+
+To ask any question, you can type your question in the text area and click on
```

Comment:
This stuff, running to the end, is fine but I'm sure it can be a bit more concise. It's kinda repetitive right now.